.github/workflows/test-all-custom-flags.yml

```diff
@@ -117,6 +118,11 @@ jobs:
           if_no_artifact_found: fail
           skip_unpack: true
 
+      - name: Set no PR feedback when missing build artifact
+        if: ${{ failure() && steps.get-build-artifact.conclusion == 'failure' }}
+        run: |
+          echo "send_pr_feedback=false" > "$GITHUB_OUTPUT"
```

Comment:
We already appended earlier, so replace instead, here (we don't use GITHUB_OUTPUT anywhere else in the job, so this should be safe)